src/tests/entities/team.test.ts

```diff
@@ -1,63 +1,147 @@
 import 'jest';
-import { UpdateResult } from 'typeorm';
+import { UpdateResult, SelectQueryBuilder } from 'typeorm';
 import { Team } from '../../entities/team';
 
 /* eslint-disable no-await-in-loop */
+const mockCreateQueryBuilder = {
+  update: jest.fn().mockReturnThis(),
+  set: jest.fn().mockReturnThis(),
+  where: jest.fn().mockReturnThis(),
+  whereInIds: jest.fn().mockReturnThis(),
+  andWhere: jest.fn().mockReturnThis(),
+  execute: jest.fn().mockReturnThis(),
+  select: jest.fn().mockReturnThis(),
+};
+const createQuerySpy = jest
+  .spyOn(Team, 'createQueryBuilder')
+  .mockReturnValue((mockCreateQueryBuilder as Partial<SelectQueryBuilder<Team>>) as SelectQueryBuilder<Team>);
 
-xdescribe('judging', () => {
-  let team: Team;
+const result: UpdateResult = {
+  affected: 0,
+  raw: undefined,
+  generatedMaps: undefined,
+};
 
-  it('decrement reduces activeJudgeCount by 1', async () => {
-    team.activeJudgeCount = 2;
-    await team.save();
-    await team.decrementActiveJudgeCount();
-    let updatedTeam = await Team.findOneOrFail(team.id);
-    expect(updatedTeam.activeJudgeCount).toEqual(1);
+const mockMethod = jest.fn(
+  (): Promise<UpdateResult> => {
+    return Promise.resolve(result);
+  },
+);
 
-    await team.decrementActiveJudgeCount();
-    updatedTeam = await Team.findOneOrFail(team.id);
-    expect(updatedTeam.activeJudgeCount).toEqual(0);
+const mockQueryBuilder = {
+  where: jest.fn().mockReturnThis(),
+  clone: jest.fn().mockReturnThis(),
+  andWhere: jest.fn().mockReturnThis(),
+  orderBy: jest.fn().mockReturnThis(),
+  addOrderBy: jest.fn().mockReturnThis(),
+  getOne: jest.fn().mockReturnThis(),
+};
+
+describe('', () => {
+  let team = new Team('Does this work?', 123, 'Databases are cool', ['123456']);
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    team.reload = jest.fn();
   });
 
-  it('activeJudgeCount will never drop below 0', async () => {
-    team.activeJudgeCount = 0;
+  it('decrement active judge count', async () => {
+    team.activeJudgeCount = 2;
+    mockCreateQueryBuilder.execute = jest.fn(() => {
+      team.activeJudgeCount = 1;
+    });
     await team.decrementActiveJudgeCount();
-    const updatedTeam = await Team.findOneOrFail(team.id);
-    expect(updatedTeam.activeJudgeCount).toEqual(0);
+    expect(createQuerySpy).toBeCalledTimes(1);
+    expect(createQuerySpy).toBeCalledWith('team');
+    expect(team.activeJudgeCount).toEqual(1);
```

Comment:
This really isn't accomplishing much... your `expect`s here are mostly just checking that the query builder was created for the right entity and that your mock method returns the value you asked it to return. Instead, we should be checking other things like `Was the set method called with an object that includes a method with that appropriate name that attempts to decrement the count?`.